cristin-project/src/main/java/no/unit/nva/cristin/projects/update/CristinProjectPatchJsonCreator.java

```diff
@@ -97,4 +113,30 @@ public ObjectNode getOutput() {
     private static List<CristinPerson> extractContributors(List<NvaContributor> contributors) {
         return contributors.stream().map(NvaContributor::toCristinPersonWithRoles).collect(Collectors.toList());
     }
+
+    private void addFundingIfPresent() {
+        if (input.has(FUNDING)) {
+            if (input.get(FUNDING).isNull()) {
+                output.putNull(PROJECT_FUNDING_SOURCES);
+            }
```

Comment:
You would need to explain the logic here to me in order for me to know why this seems weird.

Reply:
we allow nulling/emptying a field in cristin. The alternative is an empty list, which also will empty the field in cristin